catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -76,6 +88,12 @@
 
   private FeatureService featureService;
 
+  private QueryResponse queryResponse;
```

Comment:
❗️ we should not be using a field on the class for storing the queryResponse as that could lead to issues.

Instead we can use the returned response in the same scope (line 142)